mlx/linalg.cpp

```diff
@@ -0,0 +1,157 @@
+// Copyright © 2023 Apple Inc.
+
+#include <iostream>
+#include <set>
+#include <sstream>
+#include <string>
+#include <vector>
+
+#include "mlx/array.h"
+#include "mlx/linalg.h"
+#include "mlx/ops.h"
+#include "utils.h"
```

Comment:
Consistency here please `mlx/utils.h`. (I know we are not consistent in general, but let's standardize to the full path for core includes from now on.

Reply:
Of course. We should replace with 'mlx/utils.h'